packages/opencensus-core/src/stats/types.ts

```diff
@@ -14,8 +14,77 @@
  * limitations under the License.
  */
 
+import {StatsEventListener} from '../exporters/types';
 import {Metric} from '../metrics/export/types';
 
+/** Main interface for stats. */
+export interface Stats {
+  /**
+   * Creates a view.
+   * @param name The view name
+   * @param measure The view measure
+   * @param aggregation The view aggregation type
+   * @param tagKeys The view columns (tag keys)
+   * @param description The view description
+   * @param bucketBoundaries The view bucket boundaries for a distribution
+   * aggregation type
+   */
+  createView(
+      name: string, measure: Measure, aggregation: AggregationType,
+      tagKeys: string[], description: string,
+      bucketBoundaries?: number[]): View;
+
+  /**
+   * Registers a view to listen to new measurements in its measure. Prefer using
+   * the method createView() that creates an already registered view.
```

Comment:
Is this comment about creating an already registered view still true given the change above to remove the `registerView` call in the `createView` implementation above?

Reply:
Good catch, removed obsolete comment.